requirements_ray.txt

```diff
@@ -2,4 +2,4 @@ ray[default,tune]>=1.6.0
 pickle5
 tensorboardX<2.3
 GPUtil
-
+awscli
```

Comment:
Is this needed?

Reply:
It's needed for Ray to sync from s3. We can leave it out, in which case Ray will throw an exception asking for it to be installed

Reply:
Definitely fine to leave it in if needed. I wonder if this is also an issue for GCS and Azure Storage?